API/Auth.pm

```diff
@@ -59,19 +55,20 @@ sub _getCBC {
 					} else {
 						main::INFOLOG && $log->is_info && $log->info("Successfully bootstrapped") if $cbc;
 					}
-				}, sub {
+				},
+				sub {
 					Slim::Utils::Timers::setTimer(undef, time() + $cbcRetry, \&_getCBC);
 					$log->warn("Fail to get bootstrapped $url ($_[1]), retrying in $cbcRetry sec");
 					$cbcRetry *= 2;
 				}
-			)->get($url, %headers);
+			)->get($url);
 		},
 		sub {
 			Slim::Utils::Timers::setTimer(undef, time() + $cbcRetry, \&_getCBC);
 			$log->warn("Fail to get jumpstart ($_[1]), retrying in $cbcRetry sec");
 			$cbcRetry *= 2;
 		}
-	)->get('https://www.deezer.com/en/channels/explore', %headers);
+	)->get('https://www.deezer.com/en/channels/explore', 'User-Agent' => 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36');
```

Comment:
I'm sorry I wasn't clear. It's ok to define the string itself at the top. I actually prefer that. Make it a constant:

```perl
use constant DESKTOP_BROWSER => "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36";
```

What I asked you to move was only the definition of the headers object. Which with the above would become:

```perl
...->get('https://www.deezer.com/en/channels/explore', 'User-Agent' => DESKTOP_BROWSER);
```